core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooLock.java

```diff
@@ -399,9 +399,7 @@ public synchronized void process(WatchedEvent event) {
       } catch (Exception ex) {
         if (lock != null || asyncLock != null) {
           lockWatcher.unableToMonitorLockNode(ex);
-          log.error(
-              "Error resetting watch on ZooLock " + lock == null ? asyncLock : lock + " " + event,
-              ex);
+          log.error("Error resetting watch on ZooLock " + lock + " " + event, ex);
```

Comment:
This is a different log message than before if lock == null; it never logs the asyncLock info in that case.

How about:
```suggestion
          log.error("Error resetting watch on ZooLock {} {}", lock != null ? lock : asyncLock, event, ex);
```

Reply:
`lock` can be never be null due to if statement.

Reply:
I don't think that's true. One of `{lock, asyncLock}` has to be non-null, but we don't know which one, since this is the `else` portion after checking `if (event.getState() == KeeperState.Expired && lock != null)` and then the only other check is `if (lock != null || asyncLock != null)`

Reply:
You are right. I will fix.

Reply:
fixed by 8ceef50731a5fb9